quickstarts/microsoft.databricks/databricks-workspace/main.bicep

```diff
@@ -1,42 +0,0 @@
-@description('Specifies whether to deploy Azure Databricks workspace with Secure Cluster Connectivity (No Public IP) enabled or not')
```

Comment:
Just out of curiosity, is there a reason the bicep file got deleted? Was it not working as expected?